build/cqs.tsv

```diff
@@ -253,7 +253,7 @@ org.apache.spark:spark-yarn_2.11	2.2.0	provided
 org.scala-lang:scala-compiler	2.11.7	provided
 org.springframework.security:spring-security-core	4.0.4.RELEASE	provided
 org.springframework.security:spring-security-web	4.0.4.RELEASE	provided
-org.springframework:spring-web	4.2.5.RELEASE	provided
+org.springframework:spring-web	4.3.7.RELEASE	provided
```

Comment:
https://dev.eclipse.org/ipzilla/show_bug.cgi?id=15781

Reply:
@jnh5y !!!!!!!

Reply:
Yeah...  I need to get @elahrvivaz on the PMC:).

I pinged the PMC.